features/features.go

```diff
@@ -219,6 +219,14 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
+	FeatureGateCMPSMachineNamePrefix = newFeatureGate("CMPSMachineNamePrefix").
```

Comment:
There is a typo here, it would be CPMS rather than CMPS. Same issue in the other places around here.

Reply:
My bad! Thanks for the catch. I'll get them fixed in https://github.com/openshift/api/pull/2086
Copy/paste wasn't a good idea :(